modules/api-svc/src/InboundServer/handlers.js

```diff
@@ -937,12 +950,39 @@ const putBulkTransfersByIdError = async(ctx) => {
     ctx.response.body = '';
 };
 
-
 const healthCheck = async(ctx) => {
     ctx.response.status = ReturnCodes.OK.CODE;
     ctx.response.body = '';
 };
 
+const postFxQuotes = async (ctx) => {
+    const sourceFspId = ctx.request.headers['fspiop-source'];
+    const { body, headers } = ctx.request;
+    const { logger } = ctx.state;
+    const logPrefix = 'Handling POST fxQuotes request';
+
+    const model = createInboundTransfersModel(ctx);
+    model.postFxQuotes({ body, headers }, sourceFspId)
```

Comment:
You need to shallow copy the body and headers before calling the async function here. This is to avoid parallel processing issue we got previously on high concurrent calls. Please refer the functions for quotes and transfers.